frontend/src/app/home/home.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="center-text">
   <h2 lyTyp="display2" gutterBottom *ngIf="auth.loggedIn">
-    Hello, {{ (auth.userProfile) ? auth.userProfile.name : 'friend'  }}!
+    Hello, {{ (auth.userProfile) ? auth.userProfile.nickname : 'friend'  }}!
```

Comment:
This is getting moved in the next branch, will be moved into the drawer menu instead.

AKA: Click the hamburger icon -> title of drawer will have greeting for user.